modules/ROOT/pages/overview.adoc

```diff
@@ -0,0 +1,34 @@
+= Bonita Intelligent Continuous Improvement
+:page-aliases: release_notes.adoc
```

Comment:
You are using the same alias twice -> error

Reply:
removed. thanks